var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -62,9 +62,9 @@ class Llvm(CMakePackage):
     depends_on('cmake@3.4.3:', type='build')
 
     # Universal dependency
-    depends_on('python@2.7:2.8', when='@:4.999')
-    depends_on('python')
-    depends_on('py-lit', type=('build', 'run'))
```

Comment:
According to the [latest docs](https://llvm.org/docs/GettingStarted.html#software) python seems to be needed only to run tests. I assume also [py-lit](https://pypi.org/project/lit/) is needed for that purpose.